script/main.js

```diff
@@ -220,7 +222,10 @@ class App extends Component {
     // debounce just a tad, as its noisy
     debounce(_ => {
       const url = new URL(location.href);
-      const auditIdValuePairs = Object.entries(this.state).map(([id, value]) => [scoring[id].auditId,value]);
+      const auditIdValuePairs = Object.entries(this.state).map(([id, value]) => {
+        const auditId = (scoringGuides.v5[id] || scoringGuides.v6[id]).auditId;
```

Comment:
this is a bit awkward.

Reply:
yeaeh, let's make a metrics object that is just the metric name, audit id.... exactly what you named `audits`. 👍